testnets/terpnettestnet/chain.json

```diff
@@ -3,79 +3,111 @@
   "chain_name": "terpnettestnet",
   "status": "live",
   "network_type": "testnet",
-  "pretty_name": "Terp-Net",
-  "chain_id": "athena-1",
+  "pretty_name": "Terp-Network",
+  "chain_id": "athena-3",
   "bech32_prefix": "terp",
   "daemon_name": "terp",
   "node_home": "$HOME/.terp",
   
   "codebase": {
     "git_repo": "github.com/terpnetwork/terp-core.git",
-    "recommended_version": "v0.1.2",
-    "tendermint_version": "0.34.22",
+    "recommended_version": "0.2.0-3-g2b8926a",
```

Comment:
something minor, the compatible versions have the v- prefix, while the recommended version doesn't. Can we make this consistent on both?

Reply:
Updated 👌